README.md

```diff
@@ -1,2 +1,4 @@
 # HighlightPy
-A Python port of the popular Highlight bot.
+A Python port of the popular Highlight bot made by KingDGrizzle.
+
+Orignalal Bot https://github.com/KingDGrizzle/highlight
```

Comment:
1. `Orignalal` should be `Original`
2. Use proper markdown: `[Original Bot](https://github.com/KingDGrizzle/highlight)`